plugins/GSdx/GSUtil.cpp

```diff
@@ -396,6 +396,34 @@ D3D_FEATURE_LEVEL GSUtil::CheckDirect3D11Level(IDXGIAdapter *adapter, D3D_DRIVER
 	return SUCCEEDED(hr) ? level : (D3D_FEATURE_LEVEL)0;
 }
 
+GSRendererType GSUtil::GetBestRenderer()
+{
+	CComPtr<IDXGIAdapter1> adapter;
+	DXGI_ADAPTER_DESC1 desc;
+	CComPtr<IDXGIFactory1> dxgi_factory;
+
+	if (!GSUtil::CheckDXGI() || !GSUtil::CheckD3D11())
+		return GSRendererType::DX9_HW;
```

Comment:
I checked a Vista SP2 VM and dxgi.dll and d3d11.dll both exist, so the checks aren't necessary.

Notes (cause, well, you know)
1: `else` is redundant below. Up to you whether you change it or not. Of course I mention this because I prefer it without.

2: You should probably move the `adapter` and `desc` declarations closer to where they are first used.

61: I'm not really keen on the nested ifs instead of early return. IMO it's harder to read. But I suppose it hasn't quite reached the level of the first code example at https://msdn.microsoft.com/en-gb/library/windows/desktop/bb776913(v=vs.85).aspx